### ISSUE TITLE
Custom File Associations don't apply to the entire UI

### ISSUE THREAD

**Issue by neovrnaz** [ChrisRM/material-theme-jetbrains#1657]
**Description**  
When I change custom file associations, the icon in  Intellij Run/Debug Configuration stays the same.  

**Expected result**  
Change all icons associated with NPM to yarn

**Result**  
The icon associated with package.json was changed but not for Run/Debug Configuration.

**To setup a file icon association, I:**
1. Downloaded the yarn.svg from this page [File Associations](https://www.material-theme.com/docs/reference/associations/)
2. Saved it to my computers home directory
3. Add the file path `/Users/<user>/yarn.svg` to **Icon** in the material theme Custom File Associations and add the file pattern for package.json, .npmignore and .npmrc.

That works fine for the project directory, but the icon in Run/Debug Configuration is still the same as it was before, in my case "NPM". 

I'm sure that the reason is because the icon for the run/debug configuration doesn't apply to any of the files mentioned above, but It would still be nice if there was a workaround.

<img width="1162" alt="Screen Shot 2020-11-29 at 10 47 28 PM" src="https://user-images.githubusercontent.com/60867530/100576715-f19f4a00-329b-11eb-9998-e4cbdd05b88f.png">

**Environment**  
* MacOS  
* IntelliJ Idea 2020.2  
* Material Theme UI 5.3.4

**Comment by mallowigi** [comment]
Ummm yes because they are two different things. What you are referring about are "UI Icons" as opposed to "File Icons". They are icons used throughout the UI like in buttons, selectors and such.

File icons originally don't apply to UI icons, however in one of the latest releases I managed to include them as well (although this hard work got messed up by jetbrains recently), but for sure this doesn't include the custom file associations.

In short yes, custom associations (files or folders) do not change UI icons. And it's impossible to implement right now as it would demand to be able to customize each and every UI icon. Maybe one day I'll find a solution for this but this is not the priority right now.
